src/components/pages/ParentFamily/CreateNewStudent.js

```diff
@@ -29,63 +28,57 @@ const CreateNewStudent = props => {
   // };
 
   return (
-    <div className="create-new-student-modal">
-      <h2 className="create-new-student-title">Create New Student</h2>
-      <CloseOutlined className="exit" onClick={() => props.setModal(false)} />
-      <form className="create-new-student-form">
-        <label>
-          Student Name:
-          <input type="text" name="username" onChange={handleChange}></input>
-        </label>
-        <br />
-        <label>
-          Email Address:
-          <input type="text" name="email-address"></input>
-        </label>
-        <br />
-        <div>Add Prerequisites:</div>
-        <br />
-        <label>
-          HTML
-          <input
-            id={1}
-            type="checkbox"
-            name="prerequisites"
-            value="html"
-            // checked={checked.includes(checked.id)}
-            // onChange={(e) => handleChange(checked.id, e)}
-          />
-        </label>
-        <br />
-        <label>
-          CSS
-          <input
-            id={2}
-            type="checkbox"
-            name="prerequisites"
-            value="css"
-            // checked={checked.includes(checked.id)}
-            // onChange={(e) => handleChange(checked.id, e)}
-          />
-        </label>
-        <br />
-        <label>
-          JavaScript
-          <input
-            id={3}
-            type="checkbox"
-            name="prerequisites"
-            value="javascript"
-            // checked={checked.includes(checked.id)}
-            // onChange={(e) => handleChange(checked.id, e)}
-          />
-        </label>
-        <br />
-        <div className="create-new-student-submit">
-          <button type="submit">Submit</button>
-        </div>
-      </form>
-    </div>
+    <form className="create-new-student-form">
+      <label>
+        Student Name:
+        <input type="text" name="username" onChange={handleChange}></input>
+      </label>
+      <br />
+      <label>
+        Email Address:
+        <input type="text" name="email-address"></input>
+      </label>
+      <br />
+      <div>Add Prerequisites:</div>
+      <br />
+      <label>
+        HTML
+        <input
+          id={1}
+          type="checkbox"
+          name="prerequisites"
+          value="html"
+          // checked={checked.includes(checked.id)}
+          // onChange={(e) => handleChange(checked.id, e)}
+        />
+      </label>
+      <br />
+      <label>
```

Comment:
At some point, I'd like for these to be AntD components as per our standard/library use case. This will allow us to all conform to the same styling lib and standards.

Reply:
Updated! Last thing to fix is the div soup on ParentFamilyHome component. Will do tonight or Monday.